src/test/shared/credentials/userCredentialsUtils.test.ts

```diff
@@ -93,6 +93,22 @@ describe('UserCredentialsUtils', function () {
             assert(foundFiles)
             assert.strictEqual(foundFiles.length, 0)
         })
+
+        it('returns empty result if files dont contain credentials', async function () {
+            const credentialsFilename = path.join(tempFolder, 'credentials-both-exist-but-no-credentials-test')
+            const configFilename = path.join(tempFolder, 'config-both-exist-but-no-credentials-test')
+
+            const env = process.env as EnvironmentVariables
+            env.AWS_SHARED_CREDENTIALS_FILE = credentialsFilename
+            env.AWS_CONFIG_FILE = configFilename
```

Comment:
does `after()` need to restore the env?

Reply:
Yeah I'd use `sinon.replace` here with `sinon.restore` in `afterEach`

Reply:
I went with `sinon.stub` for the latest version and changed all the other tests as well. They used to remove delete the environment variables after each test